frontend/src/app/admin/PendingOrganizations/operations.graphql

```diff
@@ -0,0 +1,13 @@
+query GetUnverifiedOrganizations($identityVerified: Boolean) {
```

Comment:
The name of the query implies the argument to be false, i recommend hardcoding `identityVerified` to `false` or more generic name.